examples/data-tiering/main.tf

```diff
@@ -0,0 +1,147 @@
+provider "aws" {
+  region = local.region
+}
+
+locals {
+  region = "us-east-1"
+  name   = "memorydb-ex-${replace(basename(path.cwd), "_", "-")}"
+
+  tags = {
+    Example     = local.name
+    Environment = "dev"
+  }
+}
+
+################################################################################
+# MemoryDB Module
+################################################################################
+
+module "memory_db_disabled" {
+  source = "../.."
+
+  name   = "${local.name}-disabled"
+  create = false
+}
+
+module "memory_db" {
+  source = "../.."
+
+  # Cluster
+  name        = local.name
+  description = "Example MemoryDB cluster"
+
+  engine_version             = "7.0"
+  auto_minor_version_upgrade = true
+  node_type                  = "db.r6gd.xlarge"
+  num_shards                 = 2
+  num_replicas_per_shard     = 2
+  data_tiering               = true
```

Comment:
we can add this to the existing example - no need for a whole new example just for this

Reply:
perfect. i debated this but wasnt sure